docs/notebooks/expected_improvement.pct.py

```diff
@@ -1,5 +1,5 @@
 # %% [markdown]
-# # Introduction
+# # Two-dimensional optimization with Expected Improvement
```

Comment:
I would call it "noise-free optimization with Expected Improvement"

Reply:
done